mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -112,6 +112,12 @@ There are now three label-related fields with the new model:
 
 After upgrading to `5.0.0`, and then running `terraform refresh` or `terraform apply`, these three fields should show in the state file of the resources with a self-applying `labels` field.
 
+## Provider Default Zone and Region Values
+
+Several tests were inconsistent with one another when it came to testing `location`, `zone`, and `region` values in the provider block. The inconsistencies included some tests not working with self-links while others did. This has been addressed while also addressing the use of the `region` value within the provider block.
```

Comment:
Good point, since the tests really only effect us the maintainers and not the end-users it makes sense to just remove entirely and only mention how resources will be effected with how `region` and `zone` values are obtained now.